wasmdome/src/main.rs

```diff
@@ -63,10 +72,66 @@ fn handle_command(cmd: CliCommand) -> std::result::Result<(), Box<dyn ::std::err
             board_height,
             board_width,
         } => run_match(nc, max_turns, board_height, board_width)?,
+        WasmdomeAction::Compete { token, account } => change_compete_creds(nc, &account, &token)?,
     };
     Ok(())
 }
 
+fn change_compete_creds(
+    nc: nats::Connection,
+    account: &str,
+    token: &str,
+) -> Result<(), Box<dyn Error>> {
+    let req = CredentialsRequest {
+        account_key: account.to_string(),
+        token: token.to_string(),
+    };
+    let res = nc.request_timeout(
+        "wasmdome.public.creds.claim",
+        &serde_json::to_vec(&req)?,
+        Duration::from_millis(500),
+    )?;
+    write_arena_creds(serde_json::from_slice(&res.data)?)?;
+    Ok(())
+}
+
+fn write_arena_creds(creds: CredentialsResponse) -> Result<(), Box<dyn Error>> {
+    println!("{:?}", creds);
+    let homedir = dirs::home_dir();
+    if homedir.is_none() {
+        return Err("Cannot locate home directory".into());
+    }
+    let domedir = homedir.unwrap().join(".wasmdome/");
+    let _ = std::fs::create_dir_all(&domedir)?;
+    if let CredentialsResponse::Valid {
+        user_jwt,
+        user_secret,
+    } = creds
+    {
+        let fcontents = format!(
+            r#"
```

Comment:
I think this might need to have the tab removed, we don't want the resulting file to be indented.